File: program/program/find-the-average-of-all-numbers-in-an-array/FindTheAverageOfAllNumbersInAnArray.ts
const FindTheAverageOfAllNumbersInAnArray = (nums: number[]): number => {
    let sum: number = 0
    for (let i: number = 0; i < nums.length; i++) {
        sum += nums[i];
    }
    const avg: number = parseFloat((sum / nums.length).toFixed(2));
    console.log(avg);
    return avg;
}
export default FindTheAverageOfAllNumbersInAnArray;

File: program/program/find-the-average-of-all-numbers-in-an-array/find_the_average_of_all_numbers_in_an_array.rs
fn main(){
    let numbers = vec![1,2,3,4,5,6,7,8,9,10];
    let avg: f32;
    let mut sum: i32 = 0;
    {
        for i in &numbers{
            sum = sum + i;
        }
        avg = sum as f32 / numbers.len() as f32;
    } 
    println!("{}", avg);
}
